genie-core/src/main/java/com/netflix/genie/core/services/impl/JobCoordinatorServiceImpl.java

```diff
@@ -92,6 +93,7 @@
     private final CommandService commandService;
     private final List<ClusterLoadBalancer> clusterLoadBalancers;
     private final JobsProperties jobsProperties;
+    private final ObjectFactory<JobsUsersActiveLimitProperties> jobsUsersActiveLimitPropertiesProvider;
```

Comment:
I would have preferred `javax.inject.Provider` here to avoid vendor-specific class, but ObjectFactory is perfectly serviceable

Reply:
I used `ObjectFactory` because this is a patch for our stable for `3.3.x` branche and i choose to avoid introducing a new dependency.
When i port this change to `master` I will use `Provider`.